package.json

```diff
@@ -116,14 +118,14 @@
     "yargs": "https://registry.npmjs.org/yargs/-/yargs-3.32.0.tgz"
   },
   "dependencies": {
-    "@types/node": "^14.14.16",
     "@microsoft/ads-adal-library": "1.0.13",
     "core-js": "^2.4.1",
     "decompress-zip": "^0.2.2",
     "ejs": "^3.1.6",
     "error-ex": "^1.3.0",
     "figures": "^1.4.0",
     "find-remove": "1.2.1",
+    "fs-extra": "^10.0.0",
```

Comment:
Do we need this now with the refactoring you did?

Reply:
fs-extra is still used in serviceDownloaderProvider. If I remove that then I would also have to go and change this as well to use fs. (can be done in follow-up if necessary.